### ISSUE TITLE
Enable Jump List for pinned taskbar icon in Windows 10

### ISSUE THREAD

**Issue by krwudtke** [sobitcorp/metapad#3]
Glad to see someone is working on this great piece of software again! 
I'd love to have a working jump list when right-clicking the pinned taskbar icon for Metapad in Windows 10. In %APPDATA%\Microsoft\Windows\Recent Items, there seem to be entries for files opened by Metapad, but right-clicking the icon doesn't show any of the files. The Recent Files menu also indicates that these items are being saved, the jump list just has to be activated somehow.
Thanks

**Comment by AlisterH** [comment]
Strange, it works for me.

**Comment by krwudtke** [comment]
![image](https://user-images.githubusercontent.com/7281718/161787115-94eca9c0-e513-460b-9144-b688b764d7d9.png)
See, no files listed.

**Comment by skst** [comment]
My guess is @krwudtke is on Windows 11 and @AlisterH is on Windows 10. Microsoft rewrote the task bar for Windows 11 and left out tons of features. Jump lists, context menu, desk bands, drag and drop, etc. IMO, it's the main reason not to upgrade to Windows 11. I wish I'd known... :)

**Comment by krwudtke** [comment]
Nope, I'm on Windows 10 Pro, version 21H2, if it matters. I get jump lists on most other pinned items just fine.
![image](https://user-images.githubusercontent.com/7281718/161789998-e0396d67-828a-4473-bdce-e2b1ae1bb996.png)

**Comment by sobitcorp** [comment]
Please try right-clicking a .txt file > Open With > Choose default program > Browse...
Then find and select metapad.exe and click OK (you can also uncheck "Always use..." - it's not necessary).
